src/ntfs/write.rs

```diff
@@ -179,6 +175,48 @@ pub fn write_fares_v1(
     Ok(())
 }
 
+fn write_fares_v1_from_v2(
+    base_path: &path::Path,
+    tickets: &CollectionWithId<Ticket>,
+    ticket_uses: &CollectionWithId<TicketUse>,
+    ticket_prices: &Collection<TicketPrice>,
+    ticket_use_perimeters: &Collection<TicketUsePerimeter>,
+    ticket_use_restrictions: &Collection<TicketUseRestriction>,
+) -> Result<()> {
+    info!("xxxxxxx write_fares_v1_from_v2");
+    Ok(())
+}
+
+pub fn write_fares_v1(
```

Comment:
I think you can change the signature to something more simple: just send the entire model.
```suggestion
pub fn write_fares_v1(
	base_path: &path::Path,
	collections: &Collections,
) -> Result<()>
```